crates/ruff_python_parser/src/parser/statement.rs

```diff
@@ -890,6 +890,8 @@ impl<'src> Parser<'src> {
         let start = self.node_start();
         self.bump(TokenKind::Type);
 
+        let type_range = self.node_range(start);
+
```

Comment:
Using `node_range` to get the range of `type` is a bit confusing because it isn't a node
```suggestion
				let type_range = self.current_token_range();
        self.bump(TokenKind::Type);

```

Reply:
Thanks, I overlooked `current_token_range` before.